### PR TITLE
Feature setup associations

### PR DESCRIPTION
**Pull request by vikipretium**
> Hello 👋🏼 
> 
> ## In this step of the project:
> * `Classroom `class created and relationship between `Classroom `and `Student `implemented.
> * class `Book `created and relationship between `Person `and `Book `using the intermediate class `Rental `implemented.

Thanks for your time to review my pull request 👍🏼

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# book class
+class Book
+  attr_accessor :title, :author
```
Comment (EricMbouwe) [inline book.rb:4]: - [x] Getters and Setters for @rentals should be present to create the many-to-many relationship between the class Person and Book. Let's fix that

File: book.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rental_book(rental)
```
Comment (EricMbouwe) [inline book.rb:12]: - [x] assuming that this method is to add a new rental to the book instance, we could not find the initialisation (with date,  person) of the rental to be added to the book. Let's fix that

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @vikipretium, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book.rb (hunk 1 above)
Comment (vikipretium) [reply book.rb:4]: I'll fix it @EricMbouwe 👍🏼

File: book.rb (hunk 2 above)
Comment (vikipretium) [reply book.rb:12]: I'll fix it @EricMbouwe 👍🏼

**Review by uwadonat — APPROVED** [review]
Hi @vikipretium, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**